gen3/lib/aws.sh

```diff
@@ -280,6 +283,22 @@ EOM
     return 0
   fi
 
+  # else ...
+  if [[ "$GEN3_WORKSPACE" =~ _squidnlbcentral$ ]]; then
+    # rds snapshot vpc is simpler ...
+    commonsName=$(echo "$GEN3_WORKSPACE" | sed 's/_snapshot$//')
+    cat - <<EOM
+  env_vpc_octet3                = "3rd OCTET OF CSOC CIDR FOR SQUID SETUP"
+  env_nlb_name                  = "NLB SETUP NAME"
+  env_vpc_id                    = "CSOC VPC-ID"
+  env_pub_subnet_routetable_id = "CSOC ROUTE TABLE ID - HAVING ROUTE TO INTERNET GW"
+  csoc_internal_dns_zone_id  = "PUT IT AS `ZA1HVV5W0QBG1` IF LAUNCHING THE SQUID NLB IN CSOC MAIN VPC"
+  allowed_principals_list       = "[LIST OF AWS ACCOUNTS WHICH NEEDS TO BE WHITELISTED]"
+  # e.g. of the list - ["arn:aws:iam::<AWS ACCOUNT1 ID>:root","arn:aws:iam::<AWS ACCOUNT2 ID>:root", ...]
```

Comment:
Can we put real default values in here for our current CSOC?  We'll only be running one central squid for now, right?  :-)

Reply:
I think we better keep it this way, allowed_principal list will be modified whenever an account is added.Anyways this is launched from the csoc_master_admin VM; which anyways have all those valuse in the `.local/share/gen3/csoc/csocsquidnlb_squidnlbcentral` config.tvars file.